For Underexplored Therapeutics/Prophylactics: Interactions with Innate Immune Response (TLR agonists/antagonists & BCG trained immunity)

yemarshall opened the pull request:
I put in a paragraph on another type of possible partial-therapeutic/partial-prophylactic that is under clinical trials and also related possible prophylactics under research -- things that interact with innate immune response, mainly TLR agonists/antagonists as drafted by @rando2 at end of #349 & BCG for trained immunity clinical trials drafted by me and with feedback from @rando2 and @rlordan incorporated.
Note also, if review please also consider suggestion to expand section title of "Underexplored Therapeutics" to "Underexplored Therapeutics and Prophylactics"?

<!-- Hi there! Please use the template below as a guide for opening a pull request.
This template is designed to help the pull request process go smoother.
If anything doesn't make sense to you, please open the pull request anyway and leave entries blank as needed.
Thank you for your contribution!! -->

### Description of the proposed additions or changes
<!-- If the additions or changes aren't self explanatory feel free to leave a description or notes here. -->


### Related issues
<!-- GitHub will link issues to pull requests if you list the related issue numbers like "#2, #17". -->


### Suggested reviewers (optional)
<!-- If there are particular people you think should review this pull request, please list their GitHub handle like "@rando2".-->
@rando2 @RLordan @matfax @gregszetoAI @nilswellhausen @ajlee21 

### Checklist
<!-- Please mark the following tasks as complete by putting an "x" in the box like "[x]".
If a particular item is not applicable to your pull request, please delete the item. -->
- [ ] Text is formatted so that each sentence is on its own line.
<!-- see why this is recommended here:https://github.com/greenelab/covid19-review/blob/627a5f45a2d1fcb2150d1466df63bb848f22a3d4/USAGE.md#manubot-markdown -->
- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

yemarshall commented:
NOTE: I might remove this PR because there may be a better place to put this nestled somewhere else in the middle of the treatments section along with related ~therapeutics under research. Will look into this through #349 ...

yemarshall commented:
Another sentence to incorporate somehow since it is related to some of what's mentioned in article:
The article also mentions some data analyses which showed possible correlations between countries which have more BCG vaccination (or BCG vaccination policies) and the severity of COVID-19 in those countries, but (1) it is unclear whether this correlation indicates an interaction because for example of many other possible factors or confounding factors (country age distribution, detection efficiency, stochastic epidemic dynamic effects, differences in healthcare capacity over time in relation to epidemic dynamics, many other mundane factors) and also it is unclear what is the relation between BCG vaccination at different ages and at longer times before the start of the SARS-CoV-2 epidemic and BCG vaccination at different ages during the SARS-CoV-2 epidemic at shorter times before the risk of possible infection; at least some of these considerations are also made or pointed out in the data analyses sited and in the article.

yemarshall commented:
Thanks @rando2 for the feedback, I'll use it regardless of where this ends up in the text.

yemarshall commented:
@rando2 I'll work on incorporating your feedback -- in general, your additional/better separation of sentences/clauses will read better, but I'll need to double-check the research material again to make sure I keep the research correct while changing the separation of clauses as you suggested. I'll also check your other notes/questions.

yemarshall commented:
Although this chunk of material is still being worked on for EITHER (1) an addition within Underexplored Therapeutics OR (2) with other sort of related possible therapeutics or prophylactics [see conversation in progress at end of #349 text / 2nd file of #349], I can still get more feedback on the content. I'll make sure to incorporate it along with the general editing/clarity feedback.  I can ask more people later, but for now how about @RLordan @gregsztoAI @matfax ?

rando2 commented:
@RLordan @nilswellhausen @gregszetoAI any chance one of you might be willing to look through this? This is totally novel to me and I don't think I'll be able to offer valuable scientific critique!

RLordan commented:
> @RLordan @nilswellhausen @gregszetoAI any chance one of you might be willing to look through this? This is totally novel to me and I don't think I'll be able to offer valuable scientific critique!

Very interesting concept @yemarshall, 

I'm not an expert in the slightest on this - but two things might need to be addressed (1) is there a hypothesized mechanism for  BCG specifically by these studies or is it simply observational at this point; (2) and I think its important to report an estimation or relative risk of the infection rate/severity in BCG recipients versus non BCG people.  This paper might help with the latter, which dished out some criticism on these findings of the Spain/Italy study (https://www.medrxiv.org/content/10.1101/2020.06.05.20123539v1.full.pdf) describing it as a 'shred of evidence'. @rando2 Definitely someone more qualified than me is needed to review it thoroughly. I think it is well situated in under-explored therapeutics, although the section might be better reflected by including the word prophylactics in the section title too if BCG info remains there moving forward. 

Sorry I couldn't be more helpful.

yemarshall commented:
> @RLordan @nilswellhausen @gregszetoAI any chance one of you might be willing to look through this? This is totally novel to me and I don't think I'll be able to offer valuable scientific critique!

if you are looking at this, please also look at #481

yemarshall commented:
@RLordan thanks for the feedback. I added some mention of a few considerations/critiques for looking at some related observational studies, though the bulk of what I tried to focus on were the possible/proposed mechanisms related to "trained immunity" and the clinical trials which are interventional (not observational). I'll think about your feedback a bit more, though, because I just read it. Please also maybe look at #481 option which contains both this info & other info; you can let me know if both options are fine for organizing or if it might be better to separate it out (e.g. BCG to underexplored, but we'd probably still need to keep the other stuff especially adjuvants with the biologics if it's acceptable to the vaccine people/other reviewers of course, and maybe the mention of convalescent plasma; I thought #481 makes sense because BCG and what @rando2 put in #349 as TLR agonists/antagonists and adjuvants are related in concepts)

RLordan commented:
> @RLordan thanks for the feedback. I added some mention of a few considerations/critiques for looking at some related observational studies, though the bulk of what I tried to focus on were the possible/proposed mechanisms related to "trained immunity" and the clinical trials which are interventional (not observational). I also added types of outcomes measures such as incidence and severity or symptoms. These are in comments and in #481 and can also be added to this PR depending on which organization we choose. I'll think about your feedback a bit more, though, because I just read it. Please also maybe look at #481 option which contains both this info & other info; you can let me know if both options are fine for organizing or if it might be better to separate it out (e.g. BCG to underexplored, but we'd probably still need to keep the other stuff especially adjuvants with the biologics if it's acceptable to the vaccine people/other reviewers of course, and maybe the mention of convalescent plasma; I thought #481 makes sense because BCG and what @rando2 put in #349 as TLR agonists/antagonists and adjuvants have some overlap in concepts).
> also, for your question on mechanisms -- there are studies on BCG vaccination mechanisms related to trained immunity, many of which are reviewed in the article I cited from Nature Reviews Immunology when defining the term trained immunity.

Thanks @yemarshall , it is a really interesting area! I will look into the Nature Review :)

yemarshall commented:
Here's a version (shortened & separated out from the draft PR #481 ) which incorporates some feedback from @rando2 & @RLordan and also, because it's related, incorporates what @rando2 wrote about TLR agonists/antagonists in the 2nd part of #349 as stuff related to therapeutics under research. It'd be great to receive feedback on this either in response to this comment or once this is edited into the draft PR file:

[this can be shortened as makes sense & more generally can incorporate more feedback]

There is also some research into possible potential therapeutics or prophylactics which interact with components of the innate immune response.
Broadly, there are a variety of toll-like receptors which are examples of pattern recognition receptors (PRRs), innate immune response components which recognize pathogen-associated molecular patterns (PAMPs) and damage-associated molecular patterns (DAMPs).
Note that Toll-like receptors form a part of innate immune recognition and can more generally contribute to promoting both innate and adaptive responses.
Due to the relationship between coronaviral infections, ARDS, and the toll-like receptors TLR3 and TLR4, TLR agonists and antagonists hold some potential for broad-spectrum prophylaxis.
In mouse models, poly(I:C) and lipopolysaccharide, which are agonists of TLR3 and TLR4, respectively, showed protective effects when administered prior to SARS-CoV exposure [@doi:10.1128/JVI.01410-12].
Another type of approach which is being investigated is the possible use of what is termed trained immunity, in particular as elicited by non-SARS-CoV-2 whole-microorganism vaccines (or other microbial stimuli), as a tool which might be shown to generate heterologous protective effects with respect to SARS-CoV-2 susceptibility or severity [@doi:10.1016/j.cell.2020.04.042].
In a recent review [@doi:10.1038/s41577-020-0285-6], trained immunity was defined as forms of memory which are temporary (e.g., months or years, and reversible), displayed by innate immune cells and innate immune features of other cells, displayed as increased responsiveness to future same or heterologous pathogen infection or sometimes decreased responsiveness or immunological tolerance, and established through epigenetic and metabolic mechanisms; the review also notes various ways in which trained immunity may be related to pathology, inflammatory-related diseases, conditions and immunology associated with aging, and cancer.
One type of stimulus which research indicates can induce trained immunity is BCG vaccination (BCG is an acronym for bacillus Calmette-Guerin).
BCG is a live attenuated strain derived from an isolate of Mycobacterium bovis with the common reason for application being prevention of tuberculosis in humans.
With respect to SARS-CoV-2 specifically, clinical trials in non-SARS-CoV-2-infected adults have been setup to assess the possible efficacy of BCG vaccination, with one set of trials for example having time until first infection as the primary study endpoint, with some trials being for healthcare workers and other trials being for hospitalized elderly adults without immunosuppression who get vaccinated with placebo or BCG at hospital discharge and yet another set of trials for older adults (>50 years) under chronic care for conditions like hypertension and diabetes.
More generally, outcomes measured in some of these trials are related to incidence of disease and disease severity or symptoms.
An article [@doi:10.1016/j.cell.2020.04.042] that very briefly summarizes the setup of these trials also mentions some data analyses which showed possible correlations between countries which have more BCG vaccination (or BCG vaccination policies) and the severity of COVID-19 in those countries, but (1) it is unclear whether this correlation indicates an interaction because for example of many other possible factors or confounding factors (country age distribution, detection efficiency, stochastic epidemic dynamic effects, differences in healthcare capacity over time in relation to epidemic dynamics, many other mundane factors) and (2) also it is unclear what is the relation between BCG vaccination at different ages and at longer times before the start of the SARS-CoV-2 epidemic and BCG vaccination at different ages during the SARS-CoV-2 epidemic at shorter times before the risk of possible infection; at least some of these considerations are also made or pointed out in the data analyses sited and in the article.
The article [@doi:10.1016/j.cell.2020.04.042] also includes various related considerations (generally related animal and human studies, general safety, specific possible safety considerations with respect to SARS-CoV-2).
Additionally, we review here one of the most important considerations: severe SARS-CoV-2 has been characterized so far to possibly exhibit dysregulated immune responses and whether or when some immune responses are protective or pathogenic is still under research (e.g., [@doi:10.1016/j.chom.2020.04.009; @doi:10.1016/j.chom.2020.05.009; @doi:10.1016/j.cell.2020.04.042; @doi:10.1016/j.chom.2020.05.008]); also, trained immunity itself may sometimes be related to either balanced, or too much or too little, or useful or harmful immunological responses.
The article [@doi:10.1016/j.cell.2020.04.042] proposes that trained immunity might lead to an earlier stronger response which could reduce viremia and avoid (possibly with the additional support of another therapeutic administered later) the later detrimental immunopathology seen with severe cases; while this is a possibility, additional research is required to assess its validity.
Overlapping in these general topics, another illustration of an interesting combination of immune defense and suppression of excess inflammation with altered susceptibility to different pathogen infection is described in a review on regulation of immune response to respiratory infection [doi:10.1038/nri3166]: during the 1918 type A influenza pandemic and the 2009 H1N1 influenza pandemic, secondary bacterial pneumonia was the predominant cause of death; the possible increase and severity of secondary bacterial infections during or shortly after (or for some mechanisms weeks or months after) influenza type A infection is believed to occur because of decreased barrier function of the epithelium due to damage from the viral infection or, also, through other mechanisms in which the innate and adaptive immune cells display an altered response with respect to bacterial infection perhaps as part of immune responses regulatory features to prevent excess pulmonary inflammation along with their functions for controlling viral replication or infection.

yemarshall commented:
reminder to add once I get more feedback
reference to line 851 [ISBN: 9780815332183]

rando2 commented on content/20.treatments.md:
```diff
@@ -846,6 +846,23 @@ They may also engage the host immune response, encouraging the immune system to
+An article [@doi:10.1016/j.cell.2020.04.042] that very briefly summarizes the setup of these trials also mentions some data analyses which showed possible correlations between countries which have more BCG vaccination (or BCG vaccination policies) and the severity of COVID-19 in those countries, but (1) it is unclear whether this correlation indicates an interaction because for example of many ot ...
+The article [@doi:10.1016/j.cell.2020.04.042] also includes various related considerations (generally related animal and human studies, general safety, specific possible safety considerations with respect to SARS-CoV-2).
+Additionally, we review here one of the most important considerations: severe SARS-CoV-2 has been characterized so far to possibly exhibit dysregulated immune responses and whether or when some immune responses are protective or pathogenic is still under research (e.g., [@doi:10.1016/j.chom.2020.04.009; @doi:10.1016/j.chom.2020.05.009; @doi:10.1016/j.cell.2020.04.042; @doi:10.1016/j.chom.2020.05. ...
+The article [@doi:10.1016/j.cell.2020.04.042] proposes that trained immunity might lead to an earlier stronger response which could reduce viremia and avoid (possibly with the additional support of another therapeutic administered later) the later detrimental immunopathology seen with severe cases; while this is a possibility, additional research is required to assess its validity.
```
```suggestion
The article [@doi:10.1016/j.cell.2020.04.042] proposes that trained immunity, possibly with the subsequent administration of an additional therapeutic, might lead to an earlier and stronger response, which could reduce viremia and avoid the later detrimental immunopathology seen in severe COVID-19 cases.
While this is a possibility, additional research is required to assess its validity.
```

yemarshall replied:
I need to work on making this sentence clearer while keeping the detail of timing / different roles for different therapeutics and different stages/progression of disease... Either I find a way to write what I wrote more clearly while keeping the points I wanted, or I'll keep the original (less clear) wording. *Reminder* to keep working on making clearer.

rando2 replied:
@yemarshall I was wondering if you're still working on this? I wasn't sure based on your comments this weekend.

rando2 commented on content/20.treatments.md:
```diff
@@ -846,6 +846,23 @@ They may also engage the host immune response, encouraging the immune system to
 Upcoming work may explore the specificity of nAbs for their target, mechanisms by which the nAbs impede the virus, and improvements to antibody structure that may enhance the ability of the antibody to block viral activity.
 
+There is also some research into possible potential therapeutics or prophylactics which interact with components of the innate immune response.
+Broadly, there are a variety of toll-like receptors which are examples of pattern recognition receptors (PRRs), innate immune response components which recognize pathogen-associated molecular patterns (PAMPs) and damage-associated molecular patterns (DAMPs).
```
```suggestion
Broadly, there are a variety of toll-like receptors (TLRs) which are examples of pattern recognition receptors (PRRs), innate immune response components which recognize pathogen-associated molecular patterns (PAMPs) and damage-associated molecular patterns (DAMPs).
```

rando2 commented on content/20.treatments.md:
```diff
@@ -846,6 +846,23 @@ They may also engage the host immune response, encouraging the immune system to
 
+There is also some research into possible potential therapeutics or prophylactics which interact with components of the innate immune response.
+Broadly, there are a variety of toll-like receptors which are examples of pattern recognition receptors (PRRs), innate immune response components which recognize pathogen-associated molecular patterns (PAMPs) and damage-associated molecular patterns (DAMPs).
+Note that Toll-like receptors form a part of innate immune recognition and can more generally contribute to promoting both innate and adaptive responses.
```
```suggestion
Note that TLRs form a part of innate immune recognition and can more generally contribute to promoting both innate and adaptive responses.
```

rando2 commented on content/20.treatments.md:
```diff
@@ -846,6 +846,23 @@ They may also engage the host immune response, encouraging the immune system to
+One set of trials for example has time until first infection as the primary study endpoint; more generally, outcomes measured in some of these trials are related to incidence of disease and disease severity or symptoms.
+An article [@doi:10.1016/j.cell.2020.04.042] that very briefly summarizes the setup of these trials also mentions some data analyses which showed possible correlations between countries which have more BCG vaccination (or BCG vaccination policies) and the severity of COVID-19 in those countries, but (1) it is unclear whether this correlation indicates an interaction because for example of many ot ...
+The article [@doi:10.1016/j.cell.2020.04.042] also includes various related considerations (generally related animal and human studies, general safety, specific possible safety considerations with respect to SARS-CoV-2).
+Additionally, we review here one of the most important considerations: severe SARS-CoV-2 has been characterized so far to possibly exhibit dysregulated immune responses and whether or when some immune responses are protective or pathogenic is still under research (e.g., [@doi:10.1016/j.chom.2020.04.009; @doi:10.1016/j.chom.2020.05.009; @doi:10.1016/j.cell.2020.04.042; @doi:10.1016/j.chom.2020.05. ...
```
Note: I did not look up these papers but this seems reasonable!

yemarshall replied:
For breaking up the sentence, we can just remove the semi-colon & make it 2 sentences instead of 1 -- i think that's simpler than verifying whether our two versions of the 2nd clause are the biologically same. I like the sentence you added afterwards -- it puts additional useful points to what I wrote.
*Reminder* to partly incorporate feedback.

yemarshall commented on content/20.treatments.md:
```diff
@@ -846,6 +846,23 @@ They may also engage the host immune response, encouraging the immune system to
+In a recent review [@doi:10.1038/s41577-020-0285-6], trained immunity was defined as forms of memory which are temporary (e.g., months or years, and reversible), displayed by innate immune cells and innate immune features of other cells, displayed as increased responsiveness to future same or heterologous pathogen infection or sometimes decreased responsiveness or immunological tolerance, and est ...
+One type of stimulus which research indicates can induce trained immunity is BCG vaccination (BCG is an acronym for bacillus Calmette-Guerin).
+BCG is a live attenuated strain derived from an isolate of Mycobacterium bovis with the common reason for application being prevention of tuberculosis in humans.
+With respect to SARS-CoV-2 specifically, clinical trials in non-SARS-CoV-2-infected adults have been setup to assess the possible efficacy of BCG vaccination.
```
I chose the vague "efficacy" b/c I meant 2 ways in which it might be effective: (1) reduce susceptibility / prevent infection / ~ prophylaxis and (2) reduce disease severity / partial therapeutic. So, if you think it's better, we could write "assess BCG vaccination as a potential prophylactic and/or partial therapeutic." ??
(Note, I already mentioned in an earlier line above the other possible wording which is closer to what I saw in article "... might be shown to generate heterologous protective effects with respect to SARS-CoV-2 susceptibility or severity".)

rando2 replied:
Ah, if you would be open to the "potential prophylactic and/or partial therapeutic phrasing," I think that would really help people like me who don't know much about this!

yemarshall commented on content/20.treatments.md:
```diff
@@ -846,6 +846,23 @@ They may also engage the host immune response, encouraging the immune system to
+One type of stimulus which research indicates can induce trained immunity is BCG vaccination (BCG is an acronym for bacillus Calmette-Guerin).
+BCG is a live attenuated strain derived from an isolate of Mycobacterium bovis with the common reason for application being prevention of tuberculosis in humans.
+With respect to SARS-CoV-2 specifically, clinical trials in non-SARS-CoV-2-infected adults have been setup to assess the possible efficacy of BCG vaccination.
+Some trials enroll healthcare workers, other trials hospitalized elderly adults without immunosuppression who get vaccinated with placebo or BCG at hospital discharge, and yet another set of trials older adults (>50 years) under chronic care for conditions like hypertension and diabetes.
```
@rando2 (side-note: I didn't realize before that I had access to change the files; recently I saw how to do it, so i can do more of the editing steps now rather than asking you to do more in addition to everything else you do!)
I used present tense as a shortcut way to cover trials at different steps (those that enrolled or enrolling, those that started or still going or will start, etc. -- it's also a bit different than the therapeutic trials).
Reminder for me: I can still/will incorporate your suggestion to start the last comma clause with "and still others" instead of "and yet another set of trials"

rando2 replied:
Grammar is so hard in these contexts! My gut likes "are enrolling" but I think you're right that it implies they are still in the enrollment phase more than "enroll" does!

yemarshall commented on content/20.treatments.md:
```diff
@@ -846,6 +846,23 @@ They may also engage the host immune response, encouraging the immune system to
+BCG is a live attenuated strain derived from an isolate of Mycobacterium bovis with the common reason for application being prevention of tuberculosis in humans.
+With respect to SARS-CoV-2 specifically, clinical trials in non-SARS-CoV-2-infected adults have been setup to assess the possible efficacy of BCG vaccination.
+Some trials enroll healthcare workers, other trials hospitalized elderly adults without immunosuppression who get vaccinated with placebo or BCG at hospital discharge, and yet another set of trials older adults (>50 years) under chronic care for conditions like hypertension and diabetes.
+One set of trials for example has time until first infection as the primary study endpoint; more generally, outcomes measured in some of these trials are related to incidence of disease and disease severity or symptoms.
```
I'll use this correction except for "commonly"; "generally" works better because time until first infection is sort of related to inverse of incidence at least on some theoretical level although I'm not sure if I read the exact calculations used (I read enough to write what I wrote).

yemarshall commented on content/20.treatments.md:
```diff
@@ -846,6 +846,23 @@ They may also engage the host immune response, encouraging the immune system to
+With respect to SARS-CoV-2 specifically, clinical trials in non-SARS-CoV-2-infected adults have been setup to assess the possible efficacy of BCG vaccination.
+Some trials enroll healthcare workers, other trials hospitalized elderly adults without immunosuppression who get vaccinated with placebo or BCG at hospital discharge, and yet another set of trials older adults (>50 years) under chronic care for conditions like hypertension and diabetes.
+One set of trials for example has time until first infection as the primary study endpoint; more generally, outcomes measured in some of these trials are related to incidence of disease and disease severity or symptoms.
+An article [@doi:10.1016/j.cell.2020.04.042] that very briefly summarizes the setup of these trials also mentions some data analyses which showed possible correlations between countries which have more BCG vaccination (or BCG vaccination policies) and the severity of COVID-19 in those countries, but (1) it is unclear whether this correlation indicates an interaction because for example of many ot ...
```
i don't think "mundane" is typical outside of philosophizing english teachers or other esoteric examples ;-)

yemarshall replied:
- your re-wording of 2nd point covers everything / is more general, though I did mean something specific; I think your more wording may be better than mine because it covers several possibilities.
- i don't think "mundane" is a typical word! I sort of meant that I listed obviously possibly related factors but sometimes you also find factors that aren't obviously related, but I don't have enough experience to have an intuition or estimate of how often that is. So we can either just say "and other factors" removing the "mundane" or we can remove last comma clause and put "and" before "differences".

rando2 replied:
That sounds good to me!

yemarshall commented:
Just for a couple of PR's, I obtained feedback/review from experienced immunologist & chemist Dr. Nadia Danilova (I mainly asked her to check for any major errors & if it's pretty clear to understand; since it's just for a couple of PR's, I didn't ask her to use github and am just summarizing her feedback here). Here's the feedback for this PR:
-in general: clear, easy to understand
-suggested maybe "For example" instead of "Broadly" for start of second sentence. (Note: This works better for what I meant because other therapeutics/prophylactics discussed in review may also interact with innate immune system in other ways)
-For the following sentences:
Due to the relationship between coronaviral infections, ARDS, and the toll-like receptors TLR3 and TLR4, TLR agonists and antagonists hold some potential for broad-spectrum prophylaxis.
In mouse models, poly(I:C) and lipopolysaccharide, which are agonists of TLR3 and TLR4, respectively, showed protective effects when administered prior to SARS-CoV exposure [@doi:10.1128/JVI.01410-12].
Suggested removing first clause of first sentence above with comment that there's not enough research stated to write that there's a relationship / unclear what relationship is meant, and suggested switching places of this and the next sentence and start this sentence with Therefore, TLR agonist and antagonist...  (Note, I think it's simplest to incorporate this suggestion; otherwise, we'd need to find/put/reference a lot more research to write that there's a relationship; I had already added other related background in order to setup for these sentences and that still works to keep, so it seems that's good enough for background before going into the specific research and possible future research/applications in these sentences)
-the clause "the review also notes various ways in which trained immunity may be related to pathology, inflammatory-related diseases, conditions and immunology associated with aging, and cancer" is not relevant and should be removed. (Note, for now I separated the clause and put it in parentheses in case it might be an interesting aside -- but I will possibly/likely just remove it soon)

So, I incorporated this feedback/review along with @rando2 and @rlordan feedback (some was already incorporated before, and others I incorporated after) into the latest file for this PR.

rando2 commented on content/20.treatments.md:
```diff
@@ -897,6 +897,24 @@ They may also engage the host immune response, encouraging the immune system to
+One set of trials, for example, uses time until first infection as the primary study endpoint; more generally, outcomes measured in some of these trials are related to incidence of disease and disease severity or symptoms.
+An article [@doi:10.1016/j.cell.2020.04.042] that very briefly summarizes the setup of these trials also mentions some data analyses which showed possible correlations between countries which have more BCG vaccination (or BCG vaccination policies) and the severity of COVID-19 in those countries, but (1) it is unclear whether this correlation indicates an interaction because for example of many ot ...
+The article [@doi:10.1016/j.cell.2020.04.042] also includes various related considerations such as efficacy in related animal and human studies and the safety of the vaccine, both generally and specifically with respect to SARS-CoV-2.
+Additionally, we review here one of the most important considerations: severe SARS-CoV-2 has been characterized so far to possibly exhibit dysregulated immune responses and whether or when some immune responses are protective or pathogenic is still under research (e.g., [@doi:10.1016/j.chom.2020.04.009; @doi:10.1016/j.chom.2020.05.009; @doi:10.1016/j.cell.2020.04.042; @doi:10.1016/j.chom.2020.05. ...
```
```suggestion
Additionally, we review here one of the most important considerations: severe SARS-CoV-2 has been characterized so far to possibly exhibit dysregulated immune responses and whether or when some immune responses are protective or pathogenic is still under research (e.g., [@doi:10.1016/j.chom.2020.04.009; @doi:10.1016/j.chom.2020.05.009; @doi:10.1016/j.cell.2020.04.042; @doi:10.1016/j.chom.2020.05.008]).
Trained immunity itself may sometimes also be related to either balanced, hyperactive, or hypoactive immune responses, which can correspond to useful or harmful modulation of the immunological response.
This approach should therefore be used carefully in a disease where too weak or too strong of an immune response could both be associated with increased risk.
```
Hi @yemarshall, I tried to modify this in line with your comments above, but I think the middle sentence (after the semicolon part) does need a little bit of finessing because there are so many "or"s in a row! Perhaps something like this might help? I think breaking it up into direction (balanced, accelerated, decelerated) vs effect (harmful, useful) might help make it easier to follow.

yemarshall replied:
All the sentences are related to a very important type of consideration for COVID-19/SARS-CoV-2, so splitting it up into multiple sentences risks losing the aspect that all these points are part this type of consideration. I know grammatically it can be equivalent -- but I wanted to make sure to increase the chances that the reader gets the meaning.

yemarshall replied:
Also, though earlier I noted that I liked the sound of your last added sentence suggestion -- we already made that point in general in immunology background so I preferred to not add it here and risk making a mistake.

yemarshall commented:
@rando2 I'm not still working on this PR. What I put in this weekend for both PR's was supposed to be final draft unless there were major disagreements.

rando2 commented on content/20.treatments.md:
```diff
@@ -897,6 +897,24 @@ They may also engage the host immune response, encouraging the immune system to
+One type of stimulus which research indicates can induce trained immunity is bacillus Calmette-Guerin (BCG) vaccination.
+BCG is an attenuated form of bacteria _Mycobacterium bovis_.
+The vaccine is most commonly administered for the prevention of tuberculosis in humans.
+With respect to SARS-CoV-2 specifically, clinical trials in non-SARS-CoV-2-infected adults have been setup to assess the possible efficacy of BCG vaccination (e.g., to assess if it may be effective as a potential prophylactic or potential partial prophylactic in (1) reducing susceptibility / preventing infection and (2) reducing disease severity).
```
@cgreene caught that we don't have these clinical trials identifiers cited anywhere -- it doesn't matter whether it's here in this sentence or later in the text, but we definitely need to reference them somewhere

yemarshall replied:
I incorporated the NCT# references in the latest file & also (again) with the article reference because it describes some of those trials plus some other trials.

rando2 commented:
Thanks @yemarshall, once we add the citations to the clinical trials I think we can merge!

rando2 commented on content/20.treatments.md:
```diff
@@ -919,6 +919,24 @@ They may also engage the host immune response, encouraging the immune system to
 
+There is also some research into possible potential therapeutics or prophylactics which interact with components of the innate immune response.
+For example, there are a variety of toll-like receptors (TLRs) which are examples of pattern recognition receptors (PRRs), innate immune response components which recognize pathogen-associated molecular patterns (PAMPs) and damage-associated molecular patterns (DAMPs).
+Note that TLRs form a part of innate immune recognition and can more generally contribute to promoting both innate and adaptive responses [@ISBN: 9780815332183].
```
```suggestion
Note that TLRs form a part of innate immune recognition and can more generally contribute to promoting both innate and adaptive responses [@ISBN:9780815332183].
```
